src/webgl-utils/index.js

```diff
@@ -28,8 +28,7 @@ export {
 
 export {
   trackContextCreation,
-  createContext,
-  resizeViewport
+  createContext
```

Comment:
We typically have test files that simply check that all exports are defined. Could be good to add that.